src/main/java/org/apache/maven/plugins/dependency/analyze/AbstractAnalyzeMojo.java

```diff
@@ -231,7 +231,7 @@ public abstract class AbstractAnalyzeMojo extends AbstractMojo {
      *
      * @since 2.10
      */
-    @Parameter(defaultValue = "org.slf4j:slf4j-simple::")
+    @Parameter(defaultValue = "org.slf4j:slf4j-simple::,org.glassfish:javax.json::")
```

Comment:
How big could this list of `hidden` default exclude be? and why this javax.json specially?

Reply:
Probably less than 100 over time. Each one is an improvement.

javax.json because it's the next one I happened to stumble over when working on our own code.